spec/controllers/user_controller_spec.rb

```diff
@@ -109,6 +109,14 @@ def make_request
           not_to match(/change password, subscriptions and more/)
       end
 
+      it 'does not show private requests' do
```

Comment:
This is where the certainty of 'logged in viewing another user's requests' would be nice. Possibly also what would an admin see? (I'm guessing a standard `:admin` wouldn't see the private requests but a `:pro_admin` would?)

Reply:
Currently, only the actual user will see requests (`@is_you == true`).

This is the same for the other "personal" things – `track_things` and `undescribed_requests`.

I did think about this, but at the moment there's a _lot_ of duplication. I think we need to [review private request access permissions](https://github.com/mysociety/alaveteli-professional/issues/213) and stub out `can?` calls so that we can use the `Ability` system for this kind of testing.

Reply:
I've pushed some extra specs for viewing other users though, to cover the most important case.